PP_APP/server/frontend/templates/about.html

```diff
@@ -44,107 +44,125 @@ <h2 class="align-items-center text-center">&nbsp;</h2>
         </div>
 
 
+        <h2 class="align-items-center text-center">&nbsp;</h2>
         <h2 class="align-items-center text-center">&nbsp;</h2>
         <h2 class="align-items-center text-center">&nbsp;</h2>
         <h1 class="align-items-center text-center">OUR TEAM</h1>
         <h2 class="align-items-center text-center">&nbsp;</h2>
 
+        <div class="featured-text text-center text-lg-left">
+            <p class="text-black-80">
+                Pointless Packaging is a project developed by senior UC Davis computer
```

Comment:
I don't think the name of our project should be "Pointless Packaging", it sounds awkward. I'd just put the whole project name, "Eliminating Pointless Packaging Using Computer Vision", possibly italicize it too. That goes for all instances of the project name on the website.

Reply:
I italized it.